packages/core-mobile/app/navigation/WalletScreenStack/WalletScreenStack.tsx

```diff
@@ -240,177 +241,180 @@ function WalletScreenStack(props: Props): JSX.Element {
     <BridgeProvider>
       <NFTMetadataProvider>
         {isTestnet && <TestnetBanner />}
-        <WalletScreenS.Navigator
-          screenOptions={{
-            headerShown: false
-          }}>
-          <WalletScreenS.Screen
-            name={AppNavigation.Wallet.Drawer}
```

Comment:
<img width="1310" alt="Screenshot 2024-09-10 at 9 45 50 AM" src="https://github.com/user-attachments/assets/b8cf86a4-c0cb-450e-8a00-a9f793a2dee3">

Reply:
Should be fixed now